internal/depgraph/mod.go

```diff
@@ -0,0 +1,360 @@
+package main
+
+import (
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+	"gopkg.in/yaml.v2"
+)
+
+/*
+	This package provides a utility CLI to generate the dependency graph of each
```

Comment:
Can you use the Go doc syntax ? At the beginning of the file with `Package main provides ...`

Reply:
The first line is seen in the doc, and if you make a new line for the example, it's only when you go read the package doc.